client/web/src/components/externalServices/ExternalServiceCard.tsx

```diff
@@ -52,8 +34,15 @@ export const ExternalServiceCard: React.FunctionComponent<React.PropsWithChildre
     badge = '',
     tooltip = '',
     bordered = true,
+    status,
 }) => {
-    let cardTitle = <H3 className={shortDescription ? 'mb-0' : 'mt-1 mb-0'}>{title}</H3>
+    const titleBadge = status ? <ProductStatusBadge status={status} className="ml-2" /> : null
+    let cardTitle = (
+        <H3 className={shortDescription ? 'mb-0' : 'mt-1 mb-0'}>
+            {title}
+            {titleBadge}
```

Comment:
```suggestion
    let cardTitle = (
        <H3 className={shortDescription ? 'mb-0' : 'mt-1 mb-0'}>
            {title}
            {status && <ProductStatusBadge status={status} className="ml-2" />}
```

Nitpick, but I'd personally inline this because I think the `foo && <foo />` pattern is pretty common in our codebase :)

Reply:
Yeah, makes sense 👍